Description: Install the dma binary with the correct permissions.
 The /usr/sbin/dma binary needs to run as set-group-id to properly access
 its own mail queue.
Origin: other: http://svn.ringlet.net/svn/ringlet/mail/dma/
Forwarded: not-needed
Author: Peter Pentchev <roam@ringlet.net>
Last-Update: 2010-06-18

--- a/Makefile.plain
+++ b/Makefile.plain
@@ -30,7 +30,7 @@
 install: all
 	${INSTALL} -d ${DESTDIR}${SBIN} ${DESTDIR}${CONFDIR}
 	${INSTALL} -d ${DESTDIR}${MAN}/man8
-	${INSTALL} -m 0755 dma ${DESTDIR}${SBIN}
+	${INSTALL} -g mail -m 2755 dma ${DESTDIR}${SBIN}
 	${INSTALL} -m 0644 dma.8 ${DESTDIR}${MAN}/man8/
 
 aliases_parse.c: aliases_parse.y
